testing/run_tests.py

```diff
@@ -629,6 +629,15 @@ def ensure_ios_tests_are_built(ios_out_dir):
   assert os.path.exists(tmp_out_dir
                        ) and os.path.exists(ios_test_lib), final_message
 
+  ios_test_lib_time = os.path.getmtime(ios_test_lib)
+  flutter_dylib = os.path.join(tmp_out_dir, 'libFlutter.dylib')
+  flutter_dylib_time = os.path.getmtime(flutter_dylib)
+
+  final_message = '%s is older than %s. Please run the following commands: \n%s' % (
```

Comment:
`autoninja` is deprecated. Could you also change line 625 to be `'ninja -C ...'` instead?

Reply:
done